schemas/semconv-syntax.md

```diff
@@ -102,7 +105,12 @@ specificfields ::= spanfields
                |   metricfields
 
 spanfields ::= [events] [span_kind]
-eventfields ::= [name]
+
+eventfields ::= [name] [body]
+
+body ::= body_fields
+
+body_fields ::= id type brief [examples] stability [deprecated] [requirement_level] [note]
```

Comment:
not blocking:

I know we don't require `requirement_level` for attributes, but if I remember correctly, we wanted to start requiring it when referencing attribute.

I wonder if we can require it for events in this PR and take attributes separately.

If it's not possible in the actual implementation (because it would mean changing things for attributes), can we at least do it on the schema level?

@lquerel @jsuereth wdyt?  maybe we can discuss it on Wed tooling call.

Reply:
Agree, as nothing exists today, if we want to always enforce then this sounds like the time to make that change. Although, technically we are not "referencing" anything else today. But I agree that starting out being tighter and loosing later is easier.

Reply:
This is outside the context of this PR, but personally, I am not convinced that forcing the override of the `requirement_level` field for each `ref` on an attribute is the right approach, and this for several reasons:

- It is possible to have multiple levels of inheritance at the level of `attribute_groups`. For example, the first level of inheritance converts certain requirement levels from `opt_in` to `required`. In the following levels, we might have several branches where the `brief` / `note` of the attributes referenced in the parent level are customized based on the context for which these `attribute_group`s are intended. Saying that we force the definition of the `requirement_level` at each `ref` would imply, in this scenario, redefining the requirement level in each branch with exactly the same information, which seems suboptimal to me.
  
- More generally, I find that the `requirement_level` of an attribute is not ideally decided at the OTel registry level. It is more of a decision/directive of the end-user who, based on their own policies, might consider that a certain attribute should be required or even eliminated. This, in my view, does not advocate for making the specification of the `requirement_level` at each `ref` necessarily evident, especially for intermediate levels. In the context of multi-registry support, I would like to propose something to address this. More details on that soon.

cc @lmolkova @jsuereth

Reply:
I agree it's tricky for attributes, but here we're talking about event fields. 
We don't have a registry of them, they are not inherited or referenceable, so these problems should not(?) affect them

Reply:
BTW do we support event body inheritance? I.e. if I define

```yaml
- id: event.a
  name: event.a
  type: event
  ...
  attributes:  ...
  body:
  
- id: event.b
  name: event.b
  extend: event.a
```

what's going to happen? 

I'd prohibit it (one event extending another) for the time being. Someone always can use attribute_group to avoid copy-paste